swift_bat_position.json

```diff
@@ -0,0 +1,30 @@
+{"namespace": "gcn.notices",
+ "type": "record",
+ "name": "SWIFT_BAT_GRB_POSITION",
+ "fields": [
+     {"name": "TITLE",         "type": "string"},
+     {"name": "NOTICE_DATE",   "type": "string",          "doc": "date and time of notice creation"},
+     {"name": "NOTICE_TYPE",   "type": "string",          "doc": "type of notice"},
+     {"name": "TRIGGER_NUM",   "type": ["int", "string"], "doc": "integer or string of digits uniquely identifying event" },  
+     {"name": "SEG_NUM",       "type" : "long",           "doc": "Swift serial number (for Swift notice types only)" },
+     {"name": "GRB_RA_J2000",  "type": "double",          "doc": "ra of GRB/Transient in J2000 epoch [deg]"},
```

Comment:
* J2000 has been the preferred equinox for a long time. Do we need to specify that in the field name?
* How about just "ra", so that the field name is general to all types of transients, not just GRBs?

Reply:
This notice type is specifically only for GRBs, I think BAT would generate a different notice type if it was a non-GRB source.  Though for simplicity and comparison to other notice types, and the general use of RA/Dec, it would be easier to parse these streams if not GRB specific.